examples/denoising.jl

```diff
@@ -20,11 +20,11 @@ imgn= img .+ .01f0*randn(Float32, size(img))
 b = A*vec(imgn)
 
 # setup bregamn
-opt = bregman_options(maxIter=200, verbose=2, quantile=.5, alpha=1, antichatter=true)
-opt2 = bregman_options(maxIter=200, verbose=2, quantile=.5, alpha=1, antichatter=true, spg=true)
+opt = bregman_options(maxIter=200, verbose=2, quantile=.5, alpha=1, antichatter=true, TD=W)
```

Comment:
at that point let's just put A and x and b in options too........

Reply:
I think the bregman function itself should show `A` and `b` (or the function to calculate function value and gradient) since we are solving a linear system. In terms of the sparsifying transform ... I slightly prefer putting it into the options (since we have different "options" for sparsifying transform)